date: '2022-04-20'
sections:
  security_fixes:
    - 包已更新到最新的安全版本。 
  bugs:
    - 从存储库中删除清单文件时，不会从存储库的“依赖项关系图”页面中删除该清单。 
    - "解决了一个可能导致检索生成工件和下载 {% data variables.product.prodname_actions %} 的日志存档始终失败的回归问题。在某些情况下，我们停止解析使用 `localhost` 的内部通信的 URL，反而错误地使用了实例主机名。" 
    - 在某些情况下，使用升级包升级高可用性对中的节点可能会导致 Elasticsearch 进入不一致状态。 
    - 扩展名为 `.backup` 的轮换日志文件将累积在包含系统日志的目录中。
    - 在某些群集拓扑中，命令行实用程序 `ghe-spokesctl` 和 `ghe-btop` 无法运行。 
    - 由于 `elasticsearch-upgrade` 服务并行运行多次，因此 Elasticsearch 索引可能会在包升级期间重复。 
    - 在拉取请求和提交视图中，对于 Git LFS 跟踪的某些文件，丰富的差异将无法加载。 
    - "将用户帐户转换为组织帐户时，如果用户帐户是 {% data variables.product.prodname_ghe_server %} 企业帐户的所有者，则转换后的组织将无法正确显示在企业所有者列表中。" 
    - 当与 OAuth 应用程序 ID 匹配的集成已存在时，使用企业管理 REST API 创建模拟 OAuth 标记会导致错误。 
    - 当检测到的机密中存在 UTF8 字符时，机密扫描 REST API 将返回 `500` 响应代码。 
    - 即使数据在本地缓存位置可用，存储库缓存服务器也可以提供来自非缓存位置的数据。 
  changes:
    - 除了配置日志之外，停止 config apply 运行的配置错误现在会输出到终端。 
    - 尝试缓存大于 Memcached 中允许的最大值的值时，会引发错误，但不会报告密钥。 
    - "如果在实例上启用了 {% data variables.product.prodname_GH_advanced_security %} 功能，则在处理存储库贡献的批次时，后台作业的性能会有所提高。"
  known_issues:
    - "升级到 {% data variables.product.prodname_ghe_server %} 3.3 后，{% data variables.product.prodname_actions %} 可能无法自动启动。若要解决此问题，请通过 SSH 连接到设备并运行 `ghe-actions-start` 命令。"
    - "在新建的没有任何用户的 {% data variables.product.prodname_ghe_server %} 实例上，攻击者可以创建第一个管理员用户。"
    - 自定义防火墙规则在升级过程中被删除。
    - "[通过 Web 界面上传](https://github.com/blog/2105-upload-files-to-your-repositories)的 Git LFS 跟踪文件被错误地直接添加到存储库。"
    - 如果问题包含文件路径长于 255 个字符的同一存储库中 blob 的永久链接，则问题无法关闭。
    - "对 {% data variables.product.prodname_github_connect %} 启用“用户可以搜索 GitHub.com”后，专用和内部存储库中的问题不包括在 {% data variables.product.prodname_dotcom_the_website %} 搜索结果中。"
    - "{% data variables.product.prodname_registry %} npm 注册表不再在元数据响应中返回时间值。这样做是为了大幅改善性能。作为元数据响应的一部分，我们继续拥有返回时间值所需的所有数据，并将在我们解决现有性能问题后恢复返回这个值。"
    - 特定于处理预接收挂钩的资源限制可能导致部分预接收挂钩失败。
    - '选择“强制路径样式”时，无法验证 {% data variables.product.prodname_actions %} 存储设置并将其保存在 {% data variables.enterprise.management_console %} 中，而必须使用 `ghe-actions-precheck` 命令行实用程序进行配置。'
    - '由于当前 Linux 内核中存在 bug，安装在 Azure 上并预配了 32 个以上 CPU 内核的 {% data variables.product.prodname_ghe_server %} 3.3 实例将无法启动。[更新时间：2022-04-08]'
    - '{% data reusables.release-notes.ghas-3.4-secret-scanning-known-issue %}'
